mikupad.html

```diff
@@ -1002,6 +1003,36 @@
 	transform: translate(-50%, -50%);
 } 
 
+#button-settings {
+	display: none;
+}
+
+@media (max-width: 767.98px) {
+	body {
+		flex-direction: column;
+	}
+	#prompt-container {
+		width: 100%;
+		max-width: 100%;
+		margin-bottom: 28px;

```

Comment:
28px seems to be barely anything here, that's why I suggested dynamically changing the margin-bottom to the height of the sidebar so it would be enough for all fonts/screen sizes:
![image](https://github.com/lmg-anon/mikupad/assets/139719567/f7a8b649-41a5-4a47-80ca-9b80d660b251)